[2025-09-03T06:00:01.242+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T06:00:01.270+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_run_models.run_dbt_models scheduled__2025-09-03T05:00:00+00:00 [queued]>
[2025-09-03T06:00:01.280+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_run_models.run_dbt_models scheduled__2025-09-03T05:00:00+00:00 [queued]>
[2025-09-03T06:00:01.280+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-03T06:00:01.302+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-09-03 05:00:00+00:00
[2025-09-03T06:00:01.315+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=3054) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-03T06:00:01.317+0000] {standard_task_runner.py:64} INFO - Started process 3057 to run task
[2025-09-03T06:00:01.332+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dbt_run_models', 'run_dbt_models', 'scheduled__2025-09-03T05:00:00+00:00', '--job-id', '932', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmp0dzqasat']
[2025-09-03T06:00:01.335+0000] {standard_task_runner.py:91} INFO - Job 932: Subtask run_dbt_models
[2025-09-03T06:00:01.374+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-09-03T06:00:01.468+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_run_models.run_dbt_models scheduled__2025-09-03T05:00:00+00:00 [running]> on host b5ea13053d47
[2025-09-03T06:00:01.830+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dbt_run_models' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-03T05:00:00+00:00'
[2025-09-03T06:00:01.831+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T06:00:01.883+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-03T06:00:01.884+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        cd /usr/app &&         dbt run --profiles-dir /usr/app\n        ']
[2025-09-03T06:00:01.936+0000] {subprocess.py:86} INFO - Output:
[2025-09-03T06:00:05.766+0000] {subprocess.py:93} INFO - [0m06:00:05  Running with dbt=1.7.9
[2025-09-03T06:00:06.156+0000] {subprocess.py:93} INFO - [0m06:00:06  Registered adapter: postgres=1.7.9
[2025-09-03T06:00:06.203+0000] {subprocess.py:93} INFO - [0m06:00:06  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-09-03T06:00:06.203+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-09-03T06:00:06.203+0000] {subprocess.py:93} INFO - - models.my_project.sources
[2025-09-03T06:00:06.229+0000] {subprocess.py:93} INFO - [0m06:00:06  Found 5 models, 7 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-09-03T06:00:06.233+0000] {subprocess.py:93} INFO - [0m06:00:06
[2025-09-03T06:00:06.411+0000] {subprocess.py:93} INFO - [0m06:00:06  Concurrency: 4 threads (target='dev')
[2025-09-03T06:00:06.412+0000] {subprocess.py:93} INFO - [0m06:00:06
[2025-09-03T06:00:06.436+0000] {subprocess.py:93} INFO - [0m06:00:06  1 of 5 START sql view model analytics.stg_order_count_year ..................... [RUN]
[2025-09-03T06:00:06.441+0000] {subprocess.py:93} INFO - [0m06:00:06  2 of 5 START sql view model analytics.stg_order_details ........................ [RUN]
[2025-09-03T06:00:06.442+0000] {subprocess.py:93} INFO - [0m06:00:06  3 of 5 START sql view model analytics.stg_orders ............................... [RUN]
[2025-09-03T06:00:06.857+0000] {subprocess.py:93} INFO - [0m06:00:06  2 of 5 OK created sql view model analytics.stg_order_details ................... [[32mCREATE VIEW[0m in 0.41s]
[2025-09-03T06:00:06.878+0000] {subprocess.py:93} INFO - [0m06:00:06  1 of 5 OK created sql view model analytics.stg_order_count_year ................ [[32mCREATE VIEW[0m in 0.43s]
[2025-09-03T06:00:06.897+0000] {subprocess.py:93} INFO - [0m06:00:06  4 of 5 START sql table model analytics.order_count_year ........................ [RUN]
[2025-09-03T06:00:06.898+0000] {subprocess.py:93} INFO - [0m06:00:06  3 of 5 OK created sql view model analytics.stg_orders .......................... [[32mCREATE VIEW[0m in 0.42s]
[2025-09-03T06:00:06.936+0000] {subprocess.py:93} INFO - [0m06:00:06  5 of 5 START sql table model analytics.orders_summary .......................... [RUN]
[2025-09-03T06:00:07.350+0000] {subprocess.py:93} INFO - [0m06:00:07  4 of 5 OK created sql table model analytics.order_count_year ................... [[32mSELECT 3[0m in 0.45s]
[2025-09-03T06:00:07.591+0000] {subprocess.py:93} INFO - [0m06:00:07  5 of 5 OK created sql table model analytics.orders_summary ..................... [[32mSELECT 10000[0m in 0.65s]
[2025-09-03T06:00:07.629+0000] {subprocess.py:93} INFO - [0m06:00:07
[2025-09-03T06:00:07.630+0000] {subprocess.py:93} INFO - [0m06:00:07  Finished running 3 view models, 2 table models in 0 hours 0 minutes and 1.40 seconds (1.40s).
[2025-09-03T06:00:07.660+0000] {subprocess.py:93} INFO - [0m06:00:07
[2025-09-03T06:00:07.661+0000] {subprocess.py:93} INFO - [0m06:00:07  [32mCompleted successfully[0m
[2025-09-03T06:00:07.662+0000] {subprocess.py:93} INFO - [0m06:00:07
[2025-09-03T06:00:07.663+0000] {subprocess.py:93} INFO - [0m06:00:07  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-09-03T06:00:08.989+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-03T06:00:08.990+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T06:00:09.037+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_run_models, task_id=run_dbt_models, run_id=scheduled__2025-09-03T05:00:00+00:00, execution_date=20250903T050000, start_date=20250903T060001, end_date=20250903T060009
[2025-09-03T06:00:09.086+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-03T06:00:09.153+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-03T06:00:09.160+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
